worker/util.go

```diff
@@ -0,0 +1,52 @@
+package worker
```

Comment:
Acho que vale a pena colocar esse código dentro do pacote raiz (alba). Pode colocar ele dentro de um arquivo chamado `git.go`. Dessa forma, o pacote pipeline  não dependerá do pacote worker.

A medida que a gente for vendo como as dependências se agrupam a gente vai criando novos pacotes.

Reply:
Certo. Ajuste realizado.